src/collection-loader/validator.js

```diff
@@ -0,0 +1,39 @@
+import memoize from "lru-memoize"
+import { red } from "chalk"
+
+const fieldTypes = {
+  context: "string",
+  renderer: "function",
+  feedsOptions: "object",
+  feeds: "object",
+}
+
+const validator = (userConfig = {}) => {
+  const errors = []
+
+  Object.keys(userConfig).forEach((key) => {
+    if (!fieldTypes[key]) {
+      errors.push(
+        `Unknow option '${ key }'.`
+      )
+    }
+    else if (fieldTypes[key] !== typeof userConfig[key]) {
+      errors.push(
+        `Wrong type for '${ key }': expected '${ fieldTypes[key] }', ` +
+        `got '${ typeof userConfig[key] }'.`
+      )
+    }
+  })
+
+  if (errors.length > 0) {
+    console.error(red(
+      "Your collection loader config is invalid. Please fix the errors: \n- " +
+      errors.join("\n- ") +
+      "\n\n" +
+      "See 'Configuration' section in documentation."
+    ))
+    process.exit(0)
```

Comment:
Why not just throw an error here? I don't like using process.exit() in code other than bin.